web/files/files_test.go

```diff
@@ -3099,7 +3099,7 @@ func TestFiles(t *testing.T) {
 		// Wait for tiny thumbnail generation
 		time.Sleep(time.Second)
 
-		tiny := links.Value("tiny").String().NotEmpty().Raw()
+		// tiny := links.Value("tiny").String().NotEmpty().Raw()
```

Comment:
Shouldn't we call `t.Skip("TODO: Fix those tests")` instead of commenting out the code?

Reply:
After re-reading the doc on `t.Skip` it seems a good idea indeed. I was thinking that it would skip all the test and don't report previous errors but it's not the case. 

I change this